### PR TITLE
Update GOMA on every deployment

### PR DESCRIPTION
**Pull request by tanin47**
Apparently, we are using a super old GOMA on the CI. And this causes an issue on GOMA server. I've been told to update GOMA. Therefore, I think we should update goma on every deployment.

### CODE REVIEW

File: ci/playbook.yml [hunk 1]
```diff
@@ -64,6 +64,12 @@
       args:
         chdir: ../
         executable: /bin/bash
+    # We expect your goma to be at ~/goma.
+    - name: Update the local goma at ~/goma
+      shell: python goma_ctl.py update
+      args:
+        chdir: ~/goma
```
Comment (inferno-chromium) [inline ci/playbook.yml]: Can we use absolute paths please , like /home/goma ?

Comment (tanin47) [inline ci/playbook.yml]: It depends on the username. On my machine, it'll be `/home/tanin/goma`.

Comment (Dor1s) [inline ci/playbook.yml]: Can we use `GOMA_DIR` env var? I think it is always set up if a developer uses goma.

Comment (tanin47) [inline ci/playbook.yml]: `GOMA_DIR` on my machine is empty. But that would be more bullet-proof. I'll change to `GOMA_DIR` and default to `~/goma`.

Comment (inferno-chromium) [inline ci/playbook.yml]: SGTM

Comment (Dor1s) [inline ci/playbook.yml]: Hm, you are right, `GOMA_DIR` is not necessary. The [public doc](https://www.chromium.org/developers/gn-build-configuration) doesn't say anything about that

> Googlers can use this for distributed builds. goma_dir is only required if you put the Goma tools in a non-standard place (the default place is ~/goma or C:\goma\goma-win).

Though the internal doc suggests to use it.

**Pull request merged**